starport/pkg/cosmosclient/cosmosclient.go

```diff
@@ -384,7 +350,33 @@ func (c *Client) makeSureAccountHasTokens(ctx context.Context, address string) e
 		}
 	}
 
-	return nil
+	// make sure funds are retrieved.
+	ctx, cancel := context.WithTimeout(ctx, time.Minute)
```

Comment:
Maybe 1 minute is it not too long to wait?

Reply:
Is 5mins good?

Reply:
I'll update this 2 mins and make in configurable.

Reply:
412c8bd

Reply:
I was trying to say 1 min is too much time. If we can reduce it to ~40 seconds can be better for the user experience.

Reply:
I see now, definitely.